src/plugins/launchdarkly/ldClient.ts

```diff
@@ -9,10 +9,18 @@ export default (function () {
 
   const { LD_CLIENT_ID } = siteConfig.customFields;
 
+  const getCookie = (name: string) => {
```

Comment:
To read the cookie use `js-cookie` instead of the regex

Reply:
Good catch! Fixed & commited!